spec/menu_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'menu'
+require 'ui'
+
+describe Menu do
+  
+  let (:ui)  {instance_double(UI).as_null_object}
+  let(:menu) {Menu.new(ui)}
+
+  it "prints the menu" do
+    menuOptions = [["1", "Option 1"], ["2", "Option 2"]]
```

Comment:
This data is not important for the test.
And camel cased variable names are not idiomatic Ruby.